tests/unit/components/test_hostblock.py

```diff
@@ -570,4 +570,7 @@ def test_subdomain_blocking(config_stub, host_blocker_factory):
     config_stub.val.content.blocking.hosts.lists = None
     host_blocker = host_blocker_factory()
     host_blocker._blocked_hosts.add("example.com")
+    config_stub.val.content.blocking.hosts.block_subdomains = True
```

Comment:
Please use [parametrization](https://docs.pytest.org/en/6.2.x/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions) to turn this into two different test cases (and move this line to the other settings above).

Reply:
TIL. might be bit hackish, but I added comment about.